macos/FluentUI/Button/Button.swift

```diff
@@ -254,6 +254,39 @@ open class Button: NSButton {
 		path.fill()
 	}
 
+	public override func viewDidMoveToWindow() {
+		super.viewDidMoveToWindow()
+		isWindowInactive = !(window?.isMainWindow ?? false)
+
+		var resignMainWindowObserver: NSObjectProtocol?
+		var becomeMainWindowObserver: NSObjectProtocol?
+
```

Comment:
These are func local variables so go out of scope when the func completes. That means the `else` block below won't ever do anything